### DISCUSSION TITLE
Presentation of [nokanji] forms

### DISCUSSION THREAD

**Discussion by parfait8566** [Jitendex/Jitendex#118] [Ideas]
Firstly, wouldn't it be better to show [nokanji] forms before rare or irregular kanji forms? Unless the [nokanji] forms are also tagged as rare or irregular.
Secondly, I think putting [nokanji] forms on top with the kanji forms instead of the readings would make the table more intuitive.

Current 野次 entry:
![image](https://github.com/user-attachments/assets/2b42aa2a-d3ff-41bd-9b09-c08b72fe244d)

My suggestion:
Table | 野次 | ヤジ | 弥次 
--- | --- | --- | --- 
やじ | ◇ |  | ▽

Hovering on ヤジ could say "no associated kanji forms".

[nokanji]-like forms (コーヒー, あいつら, etc.) are fine as they are though. I guess this will make the tables not perfectly consistent, but it doesn't seem like a bad trade-off.

**Comment by stephenmk** [comment]
I appreciate the feedback. Sometimes there's something with the design of the dictionary that could obviously be improved, but it hasn't been made obvious to me yet. So it's good to get opinions from users.

I won't get rid of the ∅ columns entirely because that's how I group together multiple readings without kanji.

![kerokero](https://github.com/user-attachments/assets/7eac05ab-04a3-45dc-a5c9-b7c1ae57deab)

Replacing ∅ with the kana means that you also have to read the kana twice. This probably isn't a big issue for small entries like ヤジ, but in general I think it's easier to quickly skim and understand the meaning of the red ∅ symbol once you understand what it means.

> Firstly, wouldn't it be better to show [nokanji] forms before rare or irregular kanji forms? Unless the [nokanji] forms are also tagged as rare or irregular.

I guess there are arguments to be made for both setups, but I don't think either is clearly superior to the other. I'm happy enough with the way it is now.
